### ISSUE TITLE
`store_item<>` caused an internal panic

### ISSUE THREAD

**Issue by ryan-summers** [tweedegolf/sequential-storage#8]
When attempting to call `store_item<>`, I got an internal panic due to indexing in `sequential-storage`:
```
panicked at C:\Users\rsummers\.cargo\registry\src\index.crates.io-6f17d22bba15001f\sequential-storage-0.5.0\src\lib.rs:159:20:
range end index 32 out of range for slice of length 16
```

For reference, the calling `FlashSettings::save()` is what's doing this in the following PR https://github.com/quartiq/stabilizer/pull/811

**Comment by diondokter** [comment]
Ah! Ok, yeah that's annoying. I thought 16 bytes was a decent upper limit. I guess STM does indeed use 256-bit words. Annoying. I'll up it for the next release.

**Comment by diondokter** [comment]
Closed by #9
